Runtime/Scripts/GltFast.cs

```diff
@@ -211,8 +212,13 @@ class PrimitiveDracoCreateContext : PrimitiveCreateContextBase {
         Texture2D[] images = null;
         List<ImageCreateContext> imageCreateContexts;
 
+
         bool loadingError = false;
-        public bool LoadingError { get => loadingError; private set => loadingError = value; }
+        //this line had a compile error "only assigment, call, increment, decrement and new object expressions can be used
+        //as a statement"
+       // public bool LoadingError {get => loadingError; private set => loadingError = value; }
```

Comment:
Interesting.
Can you tell me which Unity version, platform, scripting backend, API compatibility level you used that got this error?
I'm fine with the change, but we shouldn't include the commented old code+comment.